src/qtt/pgeometry.py

```diff
@@ -1853,9 +1853,8 @@ def tilefigs(lst, geometry=[2, 2], ww=None, raisewindows=False, tofront=False,
             # assume Qt canvas
             try:
                 fig.canvas.manager.window.move(x, y)
-                fig.canvas.manager.window.resize(w, h)
-                fig.canvas.manager.window.setGeometry(x, y, w, h)
-                # mngr.window.setGeometry(x,y,w,h)
+                fig.canvas.manager.window.resize(int(w), int(h))
```

Comment:
It is not clear because there is no change there but I meant the method at line 1728 called `setWindowRectangle` also sets all kinds of stuff for the window. Here I expect the same kind of casting etc.

Reply:
I see now. The arguments to that method are the responsibility of the user. I updated the type annotations to make clear arguments should be of type `int`